webpack.config.ts

```diff
@@ -35,7 +50,16 @@ export default {
   },
   plugins: [
     new CheckerPlugin(),
+    new webpack.HotModuleReplacementPlugin(),
     new TsConfigPathsPlugin(),
+    new HtmlWebpackPlugin({
+      inject: true,
+      title: libraryName,
+      filename: 'index.html',
+      template: join(__dirname, 'src/template/common.html'),
+      hash: true,
+      chunks: ['common', 'index']
+    }),
     new TypedocWebpackPlugin(
```

Comment:
For dev we don't need to run Typedoc anymoore. Could you make it that it only runs on production?